aiplatform/src/main/java/aiplatform/vectorsearch/CreateIndexSample.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package aiplatform.vectorsearch;
+
+// [START aiplatform_vector_search_create_index_sample]
+
+import com.google.cloud.aiplatform.v1.CreateIndexRequest;
+import com.google.cloud.aiplatform.v1.Index;
+import com.google.cloud.aiplatform.v1.Index.IndexUpdateMethod;
+import com.google.cloud.aiplatform.v1.IndexServiceClient;
+import com.google.cloud.aiplatform.v1.IndexServiceSettings;
+import com.google.cloud.aiplatform.v1.LocationName;
+import com.google.protobuf.Value;
+import com.google.protobuf.util.JsonFormat;
+
+public class CreateIndexSample {
+
+  public static void main(String[] args) throws Exception {
+    // TODO(developer): Replace these variables before running the sample.
+    String project = "YOUR_PROJECT_ID";
+    String location = "YOUR_LOCATION";
+    String displayName = "YOUR_INDEX_DISPLAY_NAME";
+    String contentsDeltaUri = "gs://YOUR_BUCKET/";
+    String metadataJson = """
+        {
+          "contentsDeltaUri": "%s",
+          "config": {
+            "dimensions": 100,
+            "approximateNeighborsCount": 150,
+            "distanceMeasureType": "DOT_PRODUCT_DISTANCE",
+            "shardSize": "SHARD_SIZE_MEDIUM",
+            "algorithm_config": {
+              "treeAhConfig": {
+                "leafNodeEmbeddingCount": 5000,
+                "fractionLeafNodesToSearch": 0.03
+              }
+            }
+          }
+        }
+        """.formatted(contentsDeltaUri);
+
+    try (IndexServiceClient indexServiceClient = IndexServiceClient.create(
+        IndexServiceSettings.newBuilder().setEndpoint(location + "-aiplatform.googleapis.com:443")
+            .build())) {
+      createIndexSample(project, location, displayName, metadataJson, indexServiceClient);
+    }
+  }
+
+  static Index createIndexSample(String project, String location, String displayName,
```

Comment:
The standard format in samples is `public void return_type method(params)`

Reply:
I don't see a reason that it would be important this return `void`, as returning a value makes this easier to test. Made `public`.

Reply:
You are right. For APIs that return a value (unlike delete), it is preferred to return the value for testing. 

I missed the forward slash. `public {void / return_type} method(params)`